app/gui2/src/util/ast/reactive.ts

```diff
@@ -0,0 +1,66 @@
+import type { AstId } from 'shared/ast'
+import { type FixedMap, MutableModule } from 'shared/ast'
+import { customRef, markRaw } from 'vue'
+import * as Y from 'yjs'
+
+const WrappedValue = Symbol('wrapped')
```

Comment:
Let's make use of `Symbol.for`, so we won't have issues when HMR reloads this module. I know it kinda defeats the purpose of symbols, but the intent behind that code stays clear and we avoid annoying issues.
```suggestion
const WrappedValue = Symbol.for('Ast:WrappedValue')
```